### PR TITLE
feat: pressing cmd+j in a jazz app exports the jazz account to inspector

### PR DESCRIPTION
**Pull request by pax-k**
Usage in a Jazz app:
```
import { listenForCmdJ } from "jazz-tools";
const cleanup = listenForCmdJ();
```

On cmd+J press, redirects the user to `https://inspector.jazz.tools/import/${id}/${base64Secret}`, which in turn knows how to handle the import.

`cmd+J` listener registered for `examples/password-manager`

### CODE REVIEW

File: packages/jazz-tools/src/utils/export-account-inspector.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * @category Other
+ **/
+export function listenForCmdJ(localStorageKey?: string) {
+  if (typeof window === "undefined") return;
+
+  const cb = (e: any) => {
+    if (e.metaKey && e.key === "j") {
+      if (
+        confirm(
+          "Are you sure you want to export your account to the inspector?",
```
Comment (aeplay) [inline packages/jazz-tools/src/utils/export-account-inspector.ts]:
```suggestion
          "Are you sure you want to inspect your account using inspector.jazz.tools? This lets anyone with the secret inspector URL read your data and impersonate you.",
```

**Review by aeplay — CHANGES_REQUESTED** [review]
Left a suggestion for making the confirmation consequences clearer.

Furthermore

- Please may you put the secret into the hash part of the URL for the inspector? That way it never gets shared with the server hosting the inspector.

- Could you always enable the shortcut if the app's url contains "allowJazzInspector"? That way we can instruct users to enable it even in prod if the devs didn't

**Comment by pax-k** [comment]
> Left a suggestion for making the confirmation consequences clearer.
> 
> Furthermore
> 
> * Please may you put the secret into the hash part of the URL for the inspector? That way it never gets shared with the server hosting the inspector.
> * Could you always enable the shortcut if the app's url contains "allowJazzInspector"? That way we can instruct users to enable it even in prod if the devs didn't

Done.

If a user opens an URL like `http://localhost:5173/#/vault?allowJazzInspector`, then pressing `cmd+j` will show the confirmation and will redirect the user.

**Review by aeplay — CHANGES_REQUESTED** [review]
Thank you!

Sorry, last requests:

- please can you move the shortcut logic / function to jazz-browser? It doesn't really make sense in jazz-tools
- please add a changeset that describes the feature well for the changelog!

File: packages/jazz-browser/src/utils/export-account-inspector.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+/**
+ * Automatically sets up the Cmd+J listener if 'allowJazzInspector' is present in the URL
+ * @returns A cleanup function if the listener was set up, undefined otherwise
+ */
+export function setupInspector() {
+  if (typeof window === "undefined") return;
+
+  const url = new URL(window.location.href);
+  console.log(url.hash);
```
Comment (gdorsi) [inline packages/jazz-browser/src/utils/export-account-inspector.ts]:
```suggestion
```

File: packages/jazz-browser/src/utils/export-account-inspector.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  if (url.hash.includes("allowJazzInspector")) {
```
Comment (gdorsi) [inline packages/jazz-browser/src/utils/export-account-inspector.ts:50]: **suggestion:** what about adding `|| process.env.NODE_ENV !== 'production'` to enable this on dev?

Comment (gdorsi) [inline packages/jazz-browser/src/utils/export-account-inspector.ts:51]: **suggestion:** what about adding `|| process.env.NODE_ENV !== 'production'` to enable this on dev?

Comment (gdorsi) [inline packages/jazz-browser/src/utils/export-account-inspector.ts:50]:
```suggestion
  if (url.hash.includes("allowJazzInspector") || process.env.NODE_ENV === "development") {
```

**suggestion:** what about enabling the shortcut by default on dev?

Comment (gdorsi) [reply packages/jazz-browser/src/utils/export-account-inspector.ts:50]: cc @anselm

**Pull request merged**